webrtc.html

```diff
@@ -8064,8 +8064,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <code><a data-for="RTCDataChannel">negotiated</a></code>,
                   <code><a data-for="RTCDataChannel">id</a></code> and
                   <code><a data-for="RTCDataChannel">priority</a></code> attributes
-                  to the values of their corresponding dictionary members (if
-                  present in the dictionary).</p>
+                  to the values of their corresponding dictionary members, if
+                  present in the dictionary. For each member not present in the
+                  dictionary, initialize the corresponding attribute to its
+                  default value, as described in the attribute's description.
```

Comment:
There are only 3 members in RTCDataChannelInit that don't have default values. It would be clearer, I think, to handle them explicitly in precisely the place that an implementation would. The attribute descriptions are an odd mix of non-normative description and normative requirements, and would be nice to reduce the amount of such mixing by extracting out the normative bits.